libvirt/tests/src/backingchain/blockjob/blockjob_with_raw_option.py

```diff
@@ -0,0 +1,131 @@
+        virsh_session.sendline(cmd)
+
+    def test_raw_list():
+        """
+        Do blockjob with raw option.
+        1) Do blockcopy.
+        2) Do blockjob with raw option twice and check cur value is changing
+        3) Execute with --pivot and check raw option return empty
+        """
+        if not libvirt.check_blockjob(vm_name, dev, "progress", "100"):
+            _check_cur()
+            _abort_job()
+        else:
+            test.error('Blockcopy finished too fast, unable to check raw result,\
+             Please consider to reduce bandwith value to retest this case.')
+
+    def teardown_raw_list():
+        """
+        After blockcopy, clean file
+        """
+        # If some steps are broken by accident, execute --abort to
+        # stop blockcopy job
+        virsh.blockjob(vm_name, dev, options=' --abort', debug=True,
+                       ignore_status=True)
+
+        test_obj.clean_file(tmp_copy_path)
+        bkxml.sync()
+
+    def _check_cur():
+        """
+        Do twice blockjob --raw after blockcopy, and check the value of cur is
+        changed according to the progress of blockcopy.
+        """
+        test.log.info("TEST_STEP1: Do blockjob with --raw and check cur changed")
+        res_1 = virsh.blockjob(vm_name, dev, options=test_option, debug=True,
+                               ignore_status=False)
+        cur1 = libvirt_misc.convert_to_dict(res_1.stdout_text.strip(),
+                                            pattern=r'(\S+)=(\S+)')['cur']
+        time.sleep(1)
+        res_2 = virsh.blockjob(vm_name, dev, options=test_option, debug=True,
+                               ignore_status=False)
+        cur2 = libvirt_misc.convert_to_dict(res_2.stdout_text.strip(),
+                                            pattern=r'(\S+)=(\S+)')['cur']
+        LOG.debug('cur1 is %s , cur2 is %s', cur1, cur2)
+
+        if int(cur1) >= int(cur2):
+            test.fail('Two times cur value is not changed according'
+                      ' to the progress of blockcopy.')
+
+    def _abort_job():
+        """
+         Abort job and check abort success, and check no output after abort.
+        """
+        test.log.info("TEST_STEP2: Abort job success")
+        if utils_misc.wait_for(
+                lambda: libvirt.check_blockjob(vm_name,
+                                               dev, "progress", "100"), 100):
+            virsh.blockjob(vm_name, dev, options=' --pivot',
+                           debug=True,
+                           ignore_status=False)
+        else:
+            test.fail("Blockjob timeout in 100 sec.")
+
+        test.log.info("TEST_STEP3: Blockjob --raw no output after abort")
+        result = virsh.blockjob(vm_name, dev, options=test_option,
+                                debug=True,
+                                ignore_status=False)
+        if result.stdout_text.strip():
+            test.fail("Not return empty after pivot, but get:%s",
```

Comment:
Do you mean something like:
```suggestion
            test.fail("It should return empty after pivot, but get:%s",
```